source/php/Component/Iframe/iframe.blade.php

```diff
@@ -10,21 +10,23 @@
     ])
     <iframe id="{{ $id }}" class="{{$class}}" {!! $attribute !!}>
     </iframe>
-    @if($isSameDomain)
-        <script>
-            const acceptance    = document.querySelector('#acceptance-{{$id}}');
-            const iframe        = acceptance?.querySelector('#{{$id}}');
-            if(iframe) {
-                iframe.onload = function() {
-                    const iframeBody = iframe.contentWindow.document.body
-                    const iframeResizeObserver = new ResizeObserver(function() {
-                        const targetHeight = iframeBody.scrollHeight + 'px' || 'auto';
-                        acceptance.style.height = targetHeight;
-                        iframe.style.height = targetHeight;
-                    });
-                    iframeResizeObserver.observe(iframeBody);
+    <script>
+        window.addEventListener(
+            'message',
+            function(e) {
+                if (!event.origin.match('{{$embeddedDomain}}')) {
+                    return;
                 }
-            }
-        </script>
-    @endif
+                const acceptance    = document.querySelector('#acceptance-{{$id}}');
+                const iframe        = acceptance?.querySelector('#{{$id}}');
+                let message         = e.data;
+
+                if(iframe && acceptance && message.height) {
```

Comment:
Also check if message before checking message.height since the script will crash if the message object does not contain the high key.